src/tests/ftest/control/dmg_pool_query_test.py

```diff
@@ -84,6 +85,7 @@ def test_pool_query_basic(self):
                     "total": self.params.get("total", path="/run/exp_vals/nvme/*")
                 }
             ],
+            "dead_ranks": [],
```

Comment:
This also seems like we are making these changes in the reverse order as this entry doesn't exist in https://github.com/daos-stack/daos/blob/master/src/tests/ftest/control/dmg_pool_query_test.py